common/trainers/pit2015_trainer.py

```diff
@@ -80,6 +80,6 @@ def train(self, epochs):
 
             prev_loss = avg_loss
             if scheduler is not None:
-                scheduler.step(accuracy)
+                scheduler.step(f1)
```

Comment:
Your scheduler is monitoring f1 but saving best model (dev score) is based on accuracy, is this as intended?